### PR TITLE
[MIG] crm_deduplicate_filter > partner_deduplicate_filter: Migration to 12.0

### PR DESCRIPTION
**Pull request by victor-ditay**

### CODE REVIEW

File: partner_deduplicate_filter/README.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+#. Go to *CRM/Sales > Tools > Deduplicate Contacts*.
+#. Mark "'Is a company?' field selected", "Parent company not set" or
+   "Parent company set (Contacts)" in the section 'Exclude contacts having'.
+#. This criteria will be used for excluding in the deduplication the selected
+   kind of records.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/111/11.0
```
Comment (nikul-serpentcs) [inline partner_deduplicate_filter/README.rst]: 11.0 --> 12.0

File: partner_deduplicate_filter/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# Copyright 2017 Vicent Cubells <vicent.cubells@tecnativa.com>
+# Copyright 2019 Victor M.M Torres <victor.martin@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Exclude records from the deduplication",
+    "version": "12.0.1.0.0",
+    "category": "Tools",
+    "website": "https://www.tecnativa.com",
```
Comment (nikul-serpentcs) [inline partner_deduplicate_filter/__manifest__.py]: @victormmtorres Should be `https://github.com/OCA/partner-contact`

**Comment by victor-ditay** [comment]
@Tecnativa

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
netpicking

File: partner_deduplicate_filter/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1,9 @@
+===========================================
```
Comment (pedrobaeza) [inline partner_deduplicate_filter/readme/DESCRIPTION.rst]: This title should go out

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
@victormmtorres please remember to drop a comment when you do a change and forced push, as we are not notified in that case.

File: partner_deduplicate_filter/readme/DESCRIPTION.rst (hunk 3 above)
Comment (victor-ditay) [reply partner_deduplicate_filter/readme/DESCRIPTION.rst]: Changes done

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@nikul-serpentcs you can review again

**Comment by nikul-serpentcs** [comment]
> @nikul-serpentcs you can review again

Yes Sure

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM :+1:

**Comment by nikul-serpentcs** [comment]
@victormmtorres @pedrobaeza Squash 2 commit is more better

MIG  crm_deduplicate_filter   partner_deduplicate_filter  Migration to 12 0 by victormmtorres · Pull Request  726 · OCA partner contact

**Comment by pedrobaeza** [comment]
@nikul-serpentcs that commits are not adjacent, and there are migration commits in between, so that's not possible without conflicts.

**Pull request merged**